pkg/remotewrite/remotewrite.go

```diff
@@ -76,6 +85,50 @@ func (o *Output) Stop() error {
 	return nil
 }
 
+// setTrendStatsResolver sets the resolver for the Trend stats.
+//
+// TODO: refactor, the code can be improved
+func (o *Output) setTrendStatsResolver(trendStats []string) error {
+	var trendStatsCopy []string
+	hasSum := false
+	// copy excluding sum
+	for _, stat := range trendStats {
+		if stat == "sum" {
+			hasSum = true
+			continue
+		}
+		trendStatsCopy = append(trendStatsCopy, stat)
+	}
+	resolvers, err := metrics.GetResolversForTrendColumns(trendStatsCopy)
+	if err != nil {
+		return err
+	}
+	// sum is not supported from GetResolversForTrendColumns
+	// so if it has been requested
+	// it adds it specifically
+	if hasSum {
+		resolvers["sum"] = func(t *metrics.TrendSink) float64 {
+			return t.Sum
+		}
+	}
+	o.trendStatsResolver = make(TrendStatsResolver, len(resolvers))
+	for stat, fn := range resolvers {
+		statKey := stat
+
+		// the config passes percentiles with p(x) form, for example p(95),
+		// but the mapping generates series name in the form p95.
+		//
+		// TODO: maybe decoupling mapping from the stat resolver keys?
+		if strings.HasPrefix(statKey, "p(") {
+			statKey = stat[2 : len(statKey)-1]             // trim the parenthesis
```

Comment:
I believe you could use[ `strings.TrimPrefix`](https://pkg.go.dev/strings#TrimPrefix) here instead 👍🏻